File: substring.c
#include <stdio.h>
#include<string.h>
#include<stdlib.h>
int main()
{
    int l1,l2,i,j,t,c=0;
    char str1[50],str2[50],temp;
    scanf("%s%s",str1,str2);
    l1=strlen(str1);
    l2=strlen(str2);
    for(i=0;i<l1;i++)
    {t=i;
        for(j=0;j<l2;j++)
        {
            if(str1[t]==str2[j])
            {
                c++;t++;
                if(c==l2)
                {
                    printf("yes");
                    exit(0);
                }
            }
        }
    }//printf("%d %d\n",c,l2);
    if(c!=l2)
    printf("no");
    return 0;
}

File: substring.java
import java.util.Scanner;

public class SubString {

	public static void main(String[] args) {
		int c=1,t;
		Scanner s = new Scanner( System.in );
		String str1 = s.next();
		String str2 = s.next();
		int l1=str1.length();
		int l2=str2.length();
		for( int i = 0; i < l2; i++ ){
			t=i;
	        for( int j = 0; j < l1; j++) {
	            if( str2.charAt(t) == str1.charAt(j) ){
	            	//System.out.println(str2.charAt(t)+" "+str1.charAt(j));
	                if( c == l2 ){
	                    System.out.println("yes");
	                    System.exit(0);
	                }
	                c++;
	                t++;
	            }
	        }
	    }
	    if(c!=l2)
	    System.out.println("no");
	   

	}

}
